root/components/FormRowSelectList.js

```diff
@@ -28,54 +28,55 @@ type Props<S> = {
   +repeatable: ReadOnlyRepeatableFieldT<S>,
 };
 
-const FormRowSelectList = <S: {+id: number, ...}>({
-  addId,
-  addLabel,
-  getSelectField,
-  hideAddButton = false,
-  label,
-  onAdd,
-  onEdit,
-  onRemove,
-  options,
-  removeClassName,
-  removeLabel,
-  repeatable,
-}: Props<S>): React.Element<typeof FormRow> => (
-  <FormRow>
-    <label>{addColon(label)}</label>
-    <div className="form-row-select-list">
-      {repeatable.field.map((subfield, index) => (
-        <div className="select-list-row" key={subfield.id}>
-          <SelectField
-            field={getSelectField(subfield)}
-            onChange={event => onEdit(index, event.currentTarget.value)}
-            options={options}
-          />
-          {' '}
-          <button
-            className={`nobutton icon remove-item ${removeClassName}`}
-            onClick={() => onRemove(index)}
-            title={removeLabel}
-            type="button"
-          />
-          <FieldErrors field={getSelectField(subfield)} />
-        </div>
-      ))}
-      {hideAddButton ? null : (
-        <div className="form-row-add">
-          <button
-            className="with-label add-item"
-            id={addId}
-            onClick={onAdd}
-            type="button"
-          >
-            {addLabel}
-          </button>
-        </div>
-      )}
-    </div>
-  </FormRow>
-);
+const FormRowSelectList = <S: {+id: number, ...}>
+  ({
+    addId,
+    addLabel,
+    getSelectField,
+    hideAddButton = false,
+    label,
+    onAdd,
+    onEdit,
+    onRemove,
+    options,
+    removeClassName,
+    removeLabel,
+    repeatable,
+  }: Props<S>): React.Element<typeof FormRow> => (
+    <FormRow>
```

Comment:
This is the only way I found to make both indent and jsx-indent happy at the same time.